include/superior_mysqlpp/prepared_statements/initialize_bindings.hpp

```diff
@@ -14,14 +14,23 @@
 
 namespace SuperiorMySqlpp
 {
+    /*
+     * Objective of this whole file is proper initialization of bindings.
```

Comment:
Also please describe a little how it is actually designed. It goes a long way, when someone needs to add something. Also when and how are these initializations called.